docs/k8s.md

```diff
@@ -0,0 +1,47 @@
+# kubernetes
+
+With Inago you can easily deploy kubernetes on a coreos cluster.
+The `example` folder contains a `k8s` example. Kubernetes is split into
+two groups: `k8s-master` and `k8s-node`.
+
+## note
+The deployment sets up a kubernetes cluster, which uses insecure communication
+between the components. Use this only as a Demo or POC. If you want to deploy a
+kubernetes cluster for production please modify the example, so a secure
+communication is used. [Cluster TLS using OpenSSL](https://coreos.com/kubernetes/docs/latest/openssl.html)
+
+## deployment
+If your coreos cluster is up and running you can deploy kubernetes in 4 steps:
```

Comment:
hm, the first paragraph states that this will deploy k8s on coreos. Feels to me like that is clear enough.